### ISSUE TITLE
requests installed from .whl file will not work.

### ISSUE THREAD

**Issue by qiankunxienb** [pyscript/pyscript#112]
My Code:

```html
<html>
  <head>
    <meta charset="utf-8">
    <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>

    <py-env>
        - './requests-2.27.1-py2.py3-none-any.whl'
    </py-env>
  </head>
  <body>
    <py-script>
import requests

resp = requests.get('http://httpbin.org/ip')
print(resp.text)
    </py-script>
  </body>
</html>
```

Here, `requests-2.27.1-py2.py3-none-any.whl` is downloaded from Pypi. But when I open this html, the exception raised and it says:

<img width="1284" alt="image" src="https://user-images.githubusercontent.com/71627005/166195703-517609a3-7109-4ea9-b4ac-655e0915ae00.png">

**Comment by qiankunxienb** [comment]
The Python code work well on Jupyter.

**Comment by verhulstm** [comment]
yes, we are working on this. pyscript uses pyodide. and pyodide does not support requests yet. the browser's network security rules block many forms of networking

**Comment by verhulstm** [comment]
check out this

https://github.com/pyscript/pyscript/issues/111

and this

https://bartbroere.eu/2021/11/05/pyodide-requests/

**Comment by philippjfr** [comment]
Thanks for the issue @qiankunxienb. As @verhulstm mentioned requests is not yet supported in pyodide because the whole networking stack has to be rearchitected in the browser. This is also a duplicate of https://github.com/pyscript/pyscript/issues/91.
